article-analyzer/src/main/java/co/inventorsoft/academy/spring/config/Configuration.java

```diff
@@ -0,0 +1,33 @@
+package co.inventorsoft.academy.spring.config;
+
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import java.util.Set;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+
+/**
+ * Configuration file.
+ */
+
+@org.springframework.context.annotation.Configuration
+@ComponentScan("co.inventorsoft.academy.spring.models")
+@ComponentScan("co.inventorsoft.academy.spring.repositories")
+@ComponentScan("co.inventorsoft.academy.spring.services")
```

Comment:
is this component scan necessary?

Reply:
It was at the beginning and i forgot to remove it at the end, gonna remove it now